docs/features/voice-recording.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Voice Recording
+description:
+---
+
+Cline lets you record audio messages in chat, which are transcribed using Cline's transcription service.
+
+## How It Works
+
+1. **Enable dictation** in Feature Settings (it's on by default).
```

Comment:
We could make the docs better and more human sounding after we get a definitive decision on the approach